sdk/notificationhubs/ci.yml

```diff
@@ -25,6 +25,7 @@ pr:
     exclude:
       - sdk/servicebus/arm-notificationhubs
       - sdk/notificationhubs/ci.mgmt.yml
+      - sdk/notificationhubs/arm-notificationhubs
```

Comment:
Why do we add this line?

Reply:
@kazrael2119 Could you confirm that this is introduced by the refresh tool? If yes, pls help create an issue here https://github.com/Azure/azure-sdk-tools/issues.

Reply:
issue link:https://github.com/Azure/azure-sdk-tools/issues/4875